IanGRap/Squish
# Rewired

Goals:
- Switch input system to Rewired
- Added controller vibration in key areas

What changed:
- Installed Rewired
- Created a wrapper class around Rewired called CInput. Its a static class that models CarbonInput
- Adapted all input reading files so that they use the new system
- Added vibration calls throughout the project

How to Test:
- Play the game and make sure all of the controls still work as expected.
- Make sure you feel the controller vibrate in the following scenarios:
    - Lobby
        - When you join
        - When you leave
        - When you quit to title menu
        - When you exit the "Begin Rave" scene
        - When the floor breaks
    - Match
        - When you die
        - When lane clears
        - When bomb block goes off
    - General
        - When blocks break in camera transition
- Does the wrapper class feel clean?
- Does the vibration feel good?
